### PR TITLE
feat(extension-chrome): Scaffolding pages

### LINKED ISSUE #35: Scaffolding pages
wireframe: https://www.figma.com/file/x49RWTMPXmN5UgbaCAkraN/wallet

### PR DESCRIPTION
**Pull request by IronLu233**
This PR add add a scarfolding pages.

# Wallet manager page
For some reasons, you should manually enter the page using the URL
```
chrome-extension://{{ your extension id }}/walletManager.html
```
this page is for creating/import wallet. In the future, it can also support remove wallet?(But should we provide a way to allow user to do this **dangerous operation?**)

image

image

# Popup page
this page shows when user click icon in their Chrome browser.
image

user can manage their whitelist website, and blockchain network on the popup

https://user-images.githubusercontent.com/20639676/215243633-3253ac30-aef1-4279-9a03-46cfd7320d52.mp4
https://user-images.githubusercontent.com/20639676/215243638-c6069589-3d5c-4dac-8032-405b7993a159.mp4
closes #35

### CODE REVIEW

File: packages/extension-chrome/src/rpc/types.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export interface RpcMethods {
   wallet_enable: RpcCall<void, void>;
   wallet_isEnabled: RpcCall<void, boolean>;
   wallet_getNetworkName: RpcCall<void, string>;
+  wallet_getRandomMnemonic: RpcCall<void, string[]>;
```
Comment (IronLu233) [inline packages/extension-chrome/src/rpc/types.ts:10]: This is unnecessary. I will remove it soon

File: packages/extension-chrome/src/services/wallet.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { RPCClient as _RPCClient } from '../rpc/client';
+
+const walletService = {
+  async generateRandomMnemonic(): Promise<string[]> {
```
Comment (homura) [inline packages/extension-chrome/src/services/wallet.ts:4]: we should mark TODO if it is a mocked implementation

File: packages/extension-chrome/src/services/network.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+export type NetworkConfig = {
```
Comment (homura) [inline packages/extension-chrome/src/services/network.ts:1]: maybe it is unnecessary here, check out [ConfigService](https://github.com/ckb-js/nexus/blob/af3007977b8a24d3f91be57adf85e0866c9d6879/packages/types/src/services/ConfigService.ts)

File: packages/extension-chrome/src/services/site.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
+export type Site = {
```
Comment (homura) [inline packages/extension-chrome/src/services/site.ts:1]: maybe it is unnecessary here, check out [ConfigService](https://github.com/ckb-js/nexus/blob/af3007977b8a24d3f91be57adf85e0866c9d6879/packages/types/src/services/ConfigService.ts)

File: packages/extension-chrome/config/webpack.config.common.ts [hunk 5]
```diff
@@ -54,6 +54,7 @@ export const pageConfig: Configuration = merge(configExcludeEntry, {
   entry: {
     popup: env.paths.resolve('/src/pages/Popup/index.tsx'),
     notification: env.paths.resolve('/src/pages/Notification/index.tsx'),
+    WalletManager: env.paths.resolve('/src/pages/WalletManager/index.tsx'),
```
Comment (zhangyouxin) [inline packages/extension-chrome/config/webpack.config.common.ts:57]: maybe use only one naming style here, `CapitalCase` or `camelCase`, could be `walletManager`

**Comment by homura** [comment]
can you please make this PR to pass the CI?

File: packages/utils/src/version.ts [hunk 6]
```diff
@@ -1 +1 @@
-export const LIB_VERSION = '0.0.1';
+export const LIB_VERSION = "0.0.1";
```
[the remark this answers is not in the document]
Comment (IronLu233) [reply packages/utils/src/version.ts]: I don't know, seems my editor automatic format it unexpectedly.

**Comment by IronLu233** [comment]
> can you please make this PR to pass the CI?

OK, now they are covered.

**Review by zhangyouxin — APPROVED** [review]
LGTM

**Review by homura — APPROVED** [review]

**Comment by homura** [comment]
Conflicted, can you update the branch?

**Pull request merged**